bucket/winssh-pageant.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "1.2",
+    "description": "Pageant to Windows OpenSSH agent bridge",
+    "homepage": "https://github.com/ndbeals/winssh-pageant",
+    "license": "BSD-3-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v1.2/winssh-pageant-v1.2_amd64.zip",
+            "hash": "a9bef9d134e872e60fd1d0950ce29f73b98af4802c9e93987a4ecafe8debf814"
+        },
+        "32bit": {
+            "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v1.2/winssh-pageant-v1.2_386.zip",
+            "hash": "0591f0c4475dba8fbfacc7ffe5fcb77ac384008d884976b386b4f25fa894e3a8"
+        }
+    },
+    "extract_dir": "",
+    "bin": "winssh-pageant.exe",
+    "shortcuts": [
+        [
+            "winssh-pageant.exe",
+            "..\\Startup\\winssh-pageant"
```

Comment:
Does not make sense.

Manipulating with startup items (or any system configuraion) is awesome example of practice what scoop users are trying to avoid.

Remove whole shortcut property

Reply:
I will do this.

But this is not a tool you run when you need it (like youtube-dl or wireshark). It is a background program that's meant to run alongside the ssh-agent, very similar to [ssh-agent-wsl](https://github.com/lukesampson/scoop-extras/blob/master/bucket/ssh-agent-wsl.json). There's no direct .bashrc equivalent for outside wsl, should I use a similar `post_install` script to tell the user to create a shortcut into the appropriate startup folder?

Reply:
Use notes

Reply:
Done, thanks for the direction :)